srcpkgs/iptables/files/ip6tables/run

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
-[ ! -e /etc/iptables/ip6tables.rules ] && exit 0
-ip6tables-restore -w 3 /etc/iptables/ip6tables.rules || exit 1
+for rule in /etc/iptables/ip6tables.rules /etc/ip6tables.d/*.rules \
+    /etc/ip6tables.d/*.rules ; do
```

Comment:
duplicate glob.

Reply:
Fixed. That was the typo I mentioned in irc.